Source/WebCore/html/parser/HTMLParserIdioms.cpp

```diff
@@ -156,9 +155,8 @@ double parseToDoubleForNumberType(StringView string, double fallbackValue)
     if (!std::isfinite(value))
         return fallbackValue;
 
-    // Numbers are considered finite IEEE 754 single-precision floating point values.
-    // See HTML5 2.5.4.3 `Real numbers.'
-    if (-std::numeric_limits<float>::max() > value || value > std::numeric_limits<float>::max())
+    // Numbers are considered finite IEEE 754 Double-precision floating point values.
+    if (-std::numeric_limits<double>::max() > value || value > std::numeric_limits<double>::max())
```

Comment:
I don't think these conditions will ever hold since double can't hold values beyond min/max of double. Delete?

Reply:
Done - now running through EWS and let's see. Meanwhile I fixed other test cases so will only get "infinity" related failures.

Reply:
@rniwa - With ASSERT, the input tests are crashing in `mac-AS-debug-wk2`.

Reply:
Fixed. Thanks for pointing my silly error over Slack. Thanks!